--- libexec/NfProfile.pm	2019-05-20 16:47:09.233745030 +0600
+++ libexec/NfProfile.pm.orig	2019-05-20 16:45:47.444136227 +0600
@@ -3441,7 +3441,7 @@
 				} else {
 					$profileinfo{'type'} = 1;
 				}
-				my $status = DoRebuild(*STDOUT, \%profileinfo, $profile, $profilegroup, $profilepath, 0, 0);
+				my $status = DoRebuild(\%profileinfo, $profile, $profilegroup, $profilepath, 0, 0);
 				syslog('err', "Rebuilded profile '$profile' in group '$profilegroup': $status ");
 			}
 			if ( -f "$NfConf::PROFILESTATDIR/$profilepath/.CANCELED" ) {
@@ -3453,7 +3453,7 @@
 				} else {
 					$profileinfo{'type'} = 1;
 				}
-				my $status = DoRebuild(*STDOUT, \%profileinfo, $profile, $profilegroup, $profilepath, 0, 0);
+				my $status = DoRebuild(\%profileinfo, $profile, $profilegroup, $profilepath, 0, 0);
 				syslog('err', "Rebuilded profile '$profile' in group '$profilegroup': $status ");
 			}
 			if ( $profileinfo{'locked'} ) {
